recipe/meta.yaml

```
@@ -97,8 +97,7 @@ outputs:
         - sysroot_{{ ctng_target_platform }}
     run_exports:
       strong:
-        # The SONAME changes at major releases.
-        - {{ pin_compatible("libgfortran-ng", min_pin="x") }}
+        - libgfortran{{ libgfortran_soname }} >={{ ctng_gcc }}
```

For mutual exclusivity
```suggestion
        - libgfortran{{ libgfortran_soname }} >={{ ctng_gcc }}
        - libgfortran-ng
```